react/Radio/Readme.md

```diff
@@ -2,126 +2,132 @@
+import RadioGroup from 'cozy-ui/transpiled/react/RadioGroup'
+import FormControlLabel from 'cozy-ui/transpiled/react/FormControlLabel'
 import Variants from 'cozy-ui/docs/components/Variants'
+const Box = ({ children }) => {
+  return <div style={{ height: '3rem', width: '3rem', border: '2px dashed #CCC', display: 'flex', justifyContent: 'center', alignItems: 'center' }}>
+    { children }
+  </div>
+}
 
-let initialVariants = [
-  { error: false, disabled: false }
+initialState = { radioValue: 'item1' }
+
+const initialVariants = [
+  { checked: false, disabled: false, secondary: false }
 ]
 
-if (isTesting()) {
-  initialVariants = [
-    ...initialVariants,
-    { error: true, disabled: false },
-    { error: false, disabled: true }
-  ]
+const handleChange = event => {
+  setState({ radioValue: event.target.value  })
 }
 
-<Variants initialVariants={initialVariants}>{
-  variant => (
-    <form>
-      <div>
-        <Radio
-          name="radioForm"
-          value="radioValue1"
-          label="This is a radio button"
-          error={variant.error}
-          disabled={variant.disabled}
-        />
-        <Radio
-          name="radioForm"
-          value="radioValue2"
-          label="This is also a radio button"
-          error={variant.error}
-          disabled={variant.disabled}
-        />
+const bboxStyle = { border: '1px solid red' }
+
+;
+
+<>
+  <RadioGroup
+    aria-label="radio"
+    name="radioName"
+    value={state.radioValue.toString()}
+    onChange={handleChange}
+  >
+    <FormControlLabel
+      value="item1"
+      label="This is a radio button"
+      control={
+        <Radio />
+      }
+    />
+    <FormControlLabel
+      value="item2"
+      label="This is an intent radio button"
+      control={
+        <Radio color="secondary" />
+      }
+    />
+    <FormControlLabel
+      value="item3"
+      label="This is a disabled radio button"
+      control={
+        <Radio disabled />
+      }
+    />
+    <FormControlLabel
+      value="item4"
+      label="This is a checked disabled radio button"
+      control={
+        <Radio checked disabled />
+      }
+    />
+    <FormControlLabel
+      value="item5"
+      label="This is a edge start radio button"
+      control={
+        <Radio edge="start" />
+      }
+    />
+    <FormControlLabel
+      value="item6"
+      label="This is a edge end radio button"
+      control={
+        <Radio edge="end" />
+      }
+    />
+  </RadioGroup>
+
+  {isTesting() && (
+    <Variants initialVariants={initialVariants} screenshotAllVariants>
+      {variant => (
         <Radio
-          style={{ background: 'cadetblue',  padding: '0.5rem', height: '3rem' }}
-          name="radioForm"
-          value="radioValue3"
-          label="This is a radio button with a custom style"
-          error={variant.error}
+          color={variant.secondary ? 'secondary' : 'primary'}
+          checked={variant.checked}
           disabled={variant.disabled}
```

Comment:
can you let this one? A radio with a custom background?

Reply:
a priori ce n'est plus utile, @joel-costa ?